glance_plugin/tests/resources/test-image-start.yaml

```diff
@@ -0,0 +1,29 @@
+
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - plugin.yaml
```

Comment:
import this instead- https://raw.githubusercontent.com/cloudify-cosmo/cloudify-openstack-plugin/master/plugin.yaml